### PR TITLE
docs: Update docker rmi command

### LINKED ISSUE #9507: [BUG] Docker rmi command requires lowercase Image name
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

Local dev environment

### Version of BioDrop (for example "v1.2.3")

v2.75.0

### Description

The Docker image name in the "docker rmi" command should be in lowercase (i.e "docker rmi biodrop-app") to prevent errors, as using uppercase characters caused an error during execution. Please, refer to the attached screenshot. As, it says the name should be in lowercase, and when I changed it to lowercase, it worked. Though it's a minor issue, I want to address it. I got this error when I was trying to set up the project on my computer.

Link where I found the command in the project : https://www.biodrop.io/docs/environments/local-development#local-development

### Screenshots

Command :

Screenshot (55)

Error:

Screenshot (42)

### Do you want to work on this issue?

Yes

### If you want to work on this issue...

I'll update the command to "docker rmi biodrop-app" because the image name should to be in lowercase.

### PR DESCRIPTION
**Pull request by Shiva-Sai-ssb**
## Fixes Issue

Closes #9507 

## Changes proposed

I changed the command "docker rmi BioDrop-app" to "docker rmi biodrop-app" because the image name needs to be all in lowercase.

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: pages/docs/environments/local-development.mdx [hunk 1]
```diff
@@ -79,7 +79,7 @@ docker compose down
 
 ### Troubleshooting
 
-If you are getting an error, try to rebuild the images with `docker compose up --build`. If that doesn't work, stop the containers with `docker compose down` and remove the stopped BioDrop container with `docker rm BioDrop`. Then, remove the images with `docker rmi BioDrop-app`. Lastly, rebuild the images with `docker compose up --build`.
+If you are getting an error, try to rebuild the images with `docker compose up --build`. If that doesn't work, stop the containers with `docker compose down` and remove the stopped BioDrop container with `docker rm BioDrop`. Then, remove the images with `docker rmi biodrop-app`. Lastly, rebuild the images with `docker compose up --build`.
```
Comment (eddiejaoude) [inline pages/docs/environments/local-development.mdx:82]: Should this also be changed?

```suggestion
If you are getting an error, try to rebuild the images with `docker compose up --build`. If that doesn't work, stop the containers with `docker compose down` and remove the stopped BioDrop container with `docker rm biodrop-app`. Then, remove the images with `docker rmi biodrop-app`. Lastly, rebuild the images with `docker compose up --build`.
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you 👍 

I left an inline question about another change that looks similar

File: pages/docs/environments/local-development.mdx (hunk 1 above)
Comment (Pradumnasaraf) [reply pages/docs/environments/local-development.mdx:82]: Hey,
No, it should be `biodrop` as it's a container name and `biodrop-app` is an image name. Yes, we need to fix the capitalization from `BioDrop` to `biodrop` cc @Shiva-Sai-ssb.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

File: pages/docs/environments/local-development.mdx (hunk 1 above)
Comment (Pradumnasaraf) [reply pages/docs/environments/local-development.mdx:82]: Hey,

No, it should be `biodrop` as it's a container name and `biodrop-app` is an image name. Yes we need to fix the capitalization from `BioDrop` to `biodrop` cc @Shiva-Sai-ssb

**Comment by Shiva-Sai-ssb** [comment]
> Thank you 👍
> 
> I left an inline question about another change that looks similar

Made changes as suggested.

File: pages/docs/environments/local-development.mdx (hunk 1 above)
Comment (eddiejaoude) [reply pages/docs/environments/local-development.mdx]: Thank you Pradumna for clarifying  👍

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**